<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>kitap</class>
 <widget class="QDialog" name="kitap">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>688</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>10</y>
     <width>251</width>
     <height>41</height>
    </rect>
   </property>
   <property name="palette">
    <palette>
     <active>
      <colorrole role="WindowText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>170</red>
         <green>0</green>
         <blue>0</blue>
        </color>
       </brush>
      </colorrole>
     </active>
     <inactive>
      <colorrole role="WindowText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>170</red>
         <green>0</green>
         <blue>0</blue>
        </color>
       </brush>
      </colorrole>
     </inactive>
     <disabled/>
    </palette>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Kitap İşlemleri</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>30</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Tüm Kitaplar</string>
   </property>
  </widget>
  <widget class="QTableView" name="kitap_tw">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>60</y>
     <width>441</width>
     <height>291</height>
    </rect>
   </property>
  </widget>
  <widget class="QTableView" name="suan_tw">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>380</y>
     <width>431</width>
     <height>271</height>
    </rect>
   </property>
  </widget>
  <widget class="QTableView" name="once_tw">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>380</y>
     <width>431</width>
     <height>271</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>350</y>
     <width>321</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Seçilen Kitabın Ödünç Alınma Durumu</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>350</y>
     <width>441</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Seçilen Kitabın Daha Önceden Ödünç Alınma Durumu</string>
   </property>
  </widget>
  <widget class="QPushButton" name="kayit_btn">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>240</y>
     <width>121</width>
     <height>29</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Yeni Kayıt</string>
   </property>
  </widget>
  <widget class="QPushButton" name="guncelle_btn">
   <property name="geometry">
    <rect>
     <x>760</x>
     <y>240</y>
     <width>121</width>
     <height>29</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Güncelle</string>
   </property>
  </widget>
  <widget class="QPushButton" name="sil_btn">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>290</y>
     <width>121</width>
     <height>29</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Sil</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>60</y>
     <width>141</width>
     <height>151</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="label_5">
      <property name="font">
       <font>
        <pointsize>11</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Kitap No:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_6">
      <property name="font">
       <font>
        <pointsize>11</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Kitap Ad:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_7">
      <property name="font">
       <font>
        <pointsize>11</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Stok:</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>710</x>
     <y>60</y>
     <width>171</width>
     <height>161</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QLineEdit" name="kitap_no">
      <property name="enabled">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="kitap_ad"/>
    </item>
    <item>
     <widget class="QLineEdit" name="stok"/>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
